frontend/pulse3d/src/components/uploadForm/AnalysisParamForm.js

```diff
@@ -290,6 +297,31 @@ export default function AnalysisParamForm({
         Additional Analysis Params (Optional)
       </AdditionalParamLabel>
       <InputContainer>
+        <ParamContainer style={{ width: "33%", marginTop: "2%" }}>
+          <Label htmlFor="normalizeYAxis">
+            Disable Y-Axis Normalization :
```

Comment:
You should also remove all the negative-prefixes here as well. Also don't know if you saw my other comment but this PR needs to be merging in `RC09292022` not `main`